app/helpers/publishing_helper.rb

```diff
@@ -1,30 +1,28 @@
 module PublishingHelper
-  def handle_remote_error
+  def handle_remote_error(document)
     begin
       yield
       true
     rescue GdsApi::HTTPErrorResponse => e
-      error_response_message(:base, e.message) if @publishable
+      error_response_message(document, :base, e.message)
```

Comment:
The value of`@publishable` (a Document instance) is currently assigned in `set_errors`, and there only.

`handle_remote_error` needs this `if @publishable` because `error_response_message` calls the #errors method on it.

If we pass the Document instance to both `handle_remote_error` and `error_response_message`, I don't think we need presence check anymore.

What do you think?

Reply:
Sounds OK to me - worth a check 👍